test/apitests/service.api.test.ts

```diff
@@ -78,13 +78,6 @@ describe('features tests', () => {
     const supportTransformerProxyV1 = JSON.parse(response.text).supportTransformerProxyV1;
     expect(typeof supportTransformerProxyV1).toBe('boolean');
   });
-
```

Comment:
this test has to stay till we clean up upgradedToSourceTransformV2 variable in the system